### PR TITLE
DOCSP-40062: BsonProperty getters/setters

### PR DESCRIPTION
**Pull request by norareidy**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-40062
Staging - https://preview-mongodbnorareidy.gatsbyjs.io/java/DOCSP-40062-fix-annotation/fundamentals/data-formats/pojo-customization/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [ ] Are all the links working?
- [ ] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/data-formats/pojo-customization.txt [hunk 1]
```diff
@@ -299,9 +299,12 @@ package:
...
    * - ``BsonProperty``
-     - Specifies a custom document field name when converting the POJO
-       field to BSON. You can include a discriminator to serialize POJOs
-       nested within the field.
+     - | Specifies a custom document field name when converting the POJO
+         field to BSON. You can include a discriminator to serialize POJOs
+         nested within the field.
+       | When applying ``@BsonProperty`` to a private field, you must also add
+         getter and setter methods for that field to serialize and customize
+         the field name.
```
Comment (mcmorisi) [inline source/fundamentals/data-formats/pojo-customization.txt:307]: I notice some of the other cells in this table have a Tip box enclosed – might be a good idea to enclose the second paragraph in a Note box?

**Review by mcmorisi — APPROVED** [review]
LGTM with one comment!

File: source/fundamentals/data-formats/pojo-customization.txt (hunk 1 above)
Comment (norareidy) [reply source/fundamentals/data-formats/pojo-customization.txt:307]: I like that idea - updated!

**Review by rozza — APPROVED** [review]
LGTM

**Pull request merged**